CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+## Getting Started
+
+1. Submit an issue describing your proposed change.
```

Comment:
I think if they are adding a new repo to this list, opening an issue won't make sense.

They should only open issues if they are proposing something new. So, we can make a new section for this something like **Want to propose something?**

Reply:
actually i added that in general sense, if any person encounters any flaws or wants a feature.

Reply:
But it sounds more like they need to open an issue everytime they want to add something, even while adding a new repository.

**Feel free to open an issue if you have something new to propose** or something similar will sound better.

Reply:
yes... this should be done.. i am pushing a commit for the same